commit f3948874c340cddb8c6d93d36f35742184d601a7
Author: Takashi Sakamoto <o-takashi@sakamocchi.jp>
Date:   Mon Mar 6 12:58:14 2023 +0900

    MAINTAINERS: replace maintainer of FireWire subsystem
    
    In the last few years, I have reviewed patches for FireWire subsystem and
    requested sound subsystem maintainer to sent them to mainline, since
    FireWire subsystem maintainer has been long absent. This situation is not
    preferable since we have some user of sound hardware in IEEE 1394 bus.
    
    I will stand for the maintainer, and work for FireWire core functions and
    1394 OHCI driver, as well as sound drivers. This commit replaces the
    corresponding entry.
    
    As you know, IEEE 1394 is enough legacy. I would like to schedule the end
    of my work in the subsystem. My effort will last next 6 years. In 2026, I
    will start strong announcement for users to migrate their work load from
    IEEE 1394 bus (e.g. by purchasing alternative devices in USB and hardening
    system for them), then in 2029 let me resign the maintainer and close
    Linux 1394 project.
    
    My current work focuses on real time data (sampling data) transmission
    protocol in packet-oriented communication, thus I would provide less help
    to implementations for the other type of protocol; i.e. IPv4/IPv6 over
    IEEE 1394 bus (firewire-net), SCSI transport protocol over IEEE 1394 bus
    (firewire-sbp2) and iSCSI target (sbp-target).
    
    If receiving few objections from developers, I will start my work to send
    fixes for v6.3 prepatch, and PR for future v6.4 or later. I'm pleased if
    getting any help until the end.
    
    Reference: commit b32744751e75 ("firewire: add to MAINTAINERS")
    Acked-by: Stefan Richter <stefanr@s5r6.in-berlin.de>
    Acked-by: Takashi Iwai <tiwai@suse.de>
    Acked-by: Randy Dunlap <rdunlap@infradead.org>
    Link: https://lore.kernel.org/r/20230306035814.78455-1-o-takashi@sakamocchi.jp
    Signed-off-by: Takashi Sakamoto <o-takashi@sakamocchi.jp>

diff --git a/MAINTAINERS b/MAINTAINERS
index c6545eb54104..5cdf9f81ead6 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -7954,10 +7954,11 @@ T:	git git://git.kernel.org/pub/scm/linux/kernel/git/nab/lio-core-2.6.git master
 F:	drivers/target/sbp/
 
 FIREWIRE SUBSYSTEM
-M:	Stefan Richter <stefanr@s5r6.in-berlin.de>
+M:	Takashi Sakamoto <o-takashi@sakamocchi.jp>
+M:	Takashi Sakamoto <takaswie@kernel.org>
 L:	linux1394-devel@lists.sourceforge.net
 S:	Maintained
-W:	http://ieee1394.wiki.kernel.org/
+W:	http://ieee1394.docs.kernel.org/
 T:	git git://git.kernel.org/pub/scm/linux/kernel/git/ieee1394/linux1394.git
 F:	drivers/firewire/
 F:	include/linux/firewire.h